extension.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "RemovePII",
```

Comment:
This should be translatable as many people are not familiar with the acronym.

Reply:
> This should be translatable as many people are not familiar with the acronym.

I guess it could be. Just needs "name-msg" key in extension.json.

Reply:
Yes, that is what I am proposing changing it to. (Nitpick: it's "namemsg" not "name-msg".)

Reply:
> Yes, that is what I am proposing changing it to. (Nitpick: it's "namemsg" not "name-msg".)

Yeah I know didn't realise it put the "-" there.

Reply:
Thanks! Done.